backend/src/index.js

```diff
@@ -10,6 +10,7 @@ import subject from './modules/subject'
 import hours from './modules/hours'
 import commissions from './modules/commissions'
 import department from './modules/department'
+import semester from "./modules/semester";
```

Comment:
Don't forget to use "yarn run lint", before the push of each module. We don't use semicolons